.esh/commands/reshape_i18n_keys.py

```
@@ -0,0 +1,394 @@
+		full = f"{path}.{key}" if path else key
+		if AFFIX_RE.search(key):
+			issues.append({
+				"kind": "affix",
+				"path": full,
+				"message": f"键名「{key}」以 Suffix/Prefix 开头或结尾。{AFFIX_HINT} {UPDATE_LOCALE_DATA_HINT}",
+			})
+		if NUMBERED_RE.match(key) and not PURE_DIGITS_RE.match(key):
+			issues.append({
+				"kind": "numbered",
+				"path": full,
+				"message": f"键名「{key}」以编号结尾；请用有意义的名字，如需枚举请用数组。{UPDATE_LOCALE_DATA_HINT}",
```

Ruff (0.16.0)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)